civitai-downloader/src/core/search.py

```
@@ -0,0 +1,435 @@
+                    sort=sort,
+                    limit=limit,
+                    cursor=cursor,
+                    nsfw=search_filter.nsfw
+                )
+                
+                if not response.items:
+                    break
+                
+                # Apply client-side filters to raw data
+                # Note: We need to work with raw API data for proper filtering
+                raw_items = []  # This would need to be passed from the client
```

Ruff (0.12.2)

Local variable `raw_items` is assigned to but never used

Remove assignment to unused variable `raw_items`

(F841)